src/style.css

```diff
@@ -0,0 +1,120 @@
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 700;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 500;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+body {
+  font-weight: 300;
+  font-family: "Avenir", sans-serif;
+}
+
+.search-form {
+  margin-top: 20px;
+}
+
+.search-label {
+  position: relative;
+}
+
+.search-label::before {
+  content: "";
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  background: url(./images/Search.svg);
+  top: 0;
+  left: 26px;
+  z-index: 5;
+}
+
+.small-label::before {
+  width: 11px;
+  height: 11px;
+  bottom: 6px;
+  left: 13px;
+  background: url(./images/Search.svg);
+  background-size: contain;
+}
+
+.search-panel {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-family: "Avenir", sans-serif;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  padding: 25px 0 23px 62px;
+  transition: box-shadow 0.3s ease-in-out;
+}
+
+.search-panel::-webkit-search-decoration,
+.search-panel::-webkit-search-cancel-button,
+.search-panel::-webkit-search-results-button,
+.search-panel::-webkit-search-results-decoration {
+  display: none;
+}
+
+.search-panel::placeholder {
+  font-family: "Avenir", sans-serif;
+  color: #3d4e61;
+  font-weight: 300;
+  text-shadow: none;
+}
+
+.search-panel:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.search-panel:focus {
+  outline: none;
+  border-radius: 3px;
+  /* border: 1px solid #E1E7ED; */
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  /* box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25); */
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.small-panel {
+  font-size: 14px;
+  line-height: 19px;
+  text-shadow: none;
+  padding: 12px 0 10px 33px;
+
+}
+
+.small-panel::placeholder {
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  transition: all 0.3s ease-in-out;
+}
+
+.small-panel:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.small-panel:hover::placeholder {
+  text-shadow: none;
+}
```

Comment:
you already have `.small-panel::placeholder`

Reply:
I've resolved all issues (I hope))), except this one. My intention was to change styles on hover in .small-panel::placeholder. Are there any other ways to change it?